src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/welcome-to-premium/WelcomeToPremiumView.tsx

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styles from "./welcomeToPremium.module.scss";
+import { getL10n } from "../../../../../../../../functions/server/l10n";
+import { PercentageChart } from "../../../../../../../../components/client/PercentageChart";
+import {
+  getDashboardSummary,
+  getExposureReduction,
+} from "../../../../../../../../functions/server/dashboard";
+import { Button } from "../../../../../../../../components/server/Button";
+import {
+  StepDeterminationData,
+  getNextGuidedStep,
+} from "../../../../../../../../functions/server/getRelevantGuidedSteps";
+import { FixView } from "../../FixView";
+
+export type Props = {
+  data: StepDeterminationData;
+  subscriberEmails: string[];
+};
+
+export function WelcomeToPremiumView(props: Props) {
+  const l10n = getL10n();
+
+  const countOfDataBrokerProfiles =
+    props.data.latestScanData?.results.length ?? 0;
+  const summary = getDashboardSummary(
+    props.data.latestScanData?.results ?? [],
+    props.data.subscriberBreaches
+  );
+  const exposureReduction = getExposureReduction(
+    summary,
+    props.data.latestScanData?.results ?? []
+  );
+
+  return (
+    <FixView
+      data={props.data}
+      subscriberEmails={props.subscriberEmails}
+      nextStepHref={getNextGuidedStep(props.data, "Scan")?.href ?? ""}
+    >
+      <div className={styles.contentWrapper}>
```

Comment:
All the content in here is the same as it was in `page.tsx`, but now wrapped in `<FixView>`, and with the data passed in as props.